src/spi.rs

```diff
@@ -482,7 +482,7 @@ where
         }
 
         // Do one last status register check before continuing
-        self.check_send().ok();
+        nb::block!(self.check_send()).ok();
```

Comment:
Still `.ok()` rather than returning the result?

Reply:
I left it as `ok()` because that better preserves the previously existing behavior.

...as for whether or not it _should_ be `ok()` or `?` that is debatable.  What do you think?

I _think_ the previous intent was to clear the overrun flag if it was set, but that requires a read to DR followed by a read access to SR to clear.

Reply:
My understanding is that before it had _no_ tangible effect (since it didn't read from DR) and now it blocks but still does not affect the flags. It seems a bit strange if we were to call this function and identify a failure but not report it -- then again, if it didn't prevent transmission, perhaps it's best to ignore it. It may just depend on the error.

All in all, I don't really have an influencing perspective here since I'll be cherry-picking and tacking on my own changes in my project's fork (I need the non-blocking behavior in some cases). The `.ok()` pattern to ignore an error seems strange to me since it looks more like a mistake, but if this is the behavior that works, I can't say much against it!

Thanks for fixing this! 😄